athena-storage-api/pom.xml

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>aws-athena-query-federation</artifactId>
+        <groupId>com.amazonaws</groupId>
+        <version>2022.42.2</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>athena-storage-api</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <aws-athena-federation-sdk.version>2022.2.1</aws-athena-federation-sdk.version>
```

Comment:
This version is wrong. It should be 2022.42.2

Reply:
Changed accordingly